### ISSUE TITLE
Curious about the 'hey' message

### ISSUE THREAD

**Issue by jaewoosong** [amplab/SparkNet#51]
Good morning!

While I was running the CifarApp example, I found an interesting message. I would call it the 'hey' message. It's like:

`I0202 15:56:30.392312 16899 solver.cpp:236] Iteration 5000, loss = 1.12068`
`I0202 15:56:30.392349 16899 solver.cpp:252]     Train net output #0: loss = 1.12068 (* 1 = 1.12068 loss)`
`I0202 15:56:30.392357 16899 sgd_solver.cpp:106] Iteration 5000, lr = 0.001`
`hey`
`I0202 15:56:35.119498 16899 net.cpp:751] Copying source layer data`
`I0202 15:56:35.119524 16899 net.cpp:751] Copying source layer label`
`I0202 15:56:35.119531 16899 net.cpp:751] Copying source layer conv1`

It seems it is harmless, but may I ask what this message is and why it appears? Thank you for reading!

**Comment by robertnishihara** [comment]
Whoops! That is almost certainly a leftover print statement from some debugging. Thanks for catching that!

**Comment by jaewoosong** [comment]
Wow, thanks for the quick reply (3 minutes)! The 'hey' mystery has been solved :D
If you change the code in the future to remove the hey message, please mention me in the commit log as a mystery finder!

**Comment by javadba** [comment]
world record response time < 3 minutes.  wow that's a positive for using this project.
